Add a tad consistency in write-up.

mkcor opened the pull request:
Hi @egnha,

Great work! It is a pleasure to discover this contribution (functional love).

I'm suggesting the following minor edits to ease the reading (and be consistent with the rest of your writing).

I would also suggest merging `LICENSE` and `LICENSE.md` into a single file (probably into `LICENSE` since it is the filename you end up with when creating a package with `devtools`), knowing that the information is even present in the README...

Best,
Marianne

mkcor committed:
Add a tad consistency in write-up.

egnha commented:
Thanks for the kind words and edits! :-)

Could you kindly edit README.Rmd, and use knitr to generate README.md from that?

As for LICENSE/LICENSE.md, I agree that it seems redundant to have both. :-/ Unless I'm mistaken, I believe LICENSE is required in its current form. But LICENSE.md could probably be merged into LICENSE (presuming the format permits that), or simply omitted. Thanks for the suggestion.

mkcor committed:
Revert "Add a tad consistency in write-up."

This reverts commit 1e067d56dc9d4fc3c9eade15695fba9addc5abe8.

mkcor committed:
Edit README in source file.

mkcor committed:
Remove extra LICENSE file.

mkcor commented:
Oh, I didn't realize that `README.md is generated from README.Rmd. Please edit that file` ;)
I have reverted my initial commit (I thought of rebasing, but we might as well keep track of everything that happened) and edited `README.Rmd` instead.

Right, you want to keep `LICENSE` in its current form (reference: http://r-pkgs.had.co.nz/description.html#license). `LICENSE.md` is definitely redundant (we know the license is MIT from `DESCRIPTION` as well as from the READMEs). So, I have added a commit with removes it.

egnha commented:
Thanks, again!
